### PR TITLE
[Feature] Support YOLOv5 deployment at RV1126 device

### PR DESCRIPTION
**Pull request by AllentDan**

### CODE REVIEW

File: configs/deploy/detection_rknn_static-320x320.py
Comment (triple-Mu) [inline configs/deploy/detection_rknn_static-320x320.py:4]:
```suggestion
```

File: mmyolo/deploy/models/dense_heads/yolov5_head.py
Comment (PeterH0323) [inline mmyolo/deploy/models/dense_heads/yolov5_head.py:155]:
```suggestion
def yolov5_head__predict__rknn(ctx, self, x: Tuple[Tensor], *args, **kwargs) -> Tensor:
```

File: mmyolo/deploy/models/dense_heads/yolov5_head.py
Comment (PeterH0323) [inline mmyolo/deploy/models/dense_heads/yolov5_head.py:156]: maybe something wrong after the merge 😢

Comment (AllentDan) [reply mmyolo/deploy/models/dense_heads/yolov5_head.py]: Not familiar with `Allow edits by maintainers`, closed now.

**Comment by triple-Mu** [comment]
LGTM

**Review by triple-Mu — APPROVED** [review]
LGTM

**Review by PeterH0323 — APPROVED** [review]
LGTM

**Comment by PeterH0323** [comment]
Hi @AllentDan 

Pls fix the ci error

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**
